homeassistant/components/zwave_js/api.py

```diff
@@ -1335,24 +1308,21 @@ async def websocket_stop_healing_network(
 @websocket_api.websocket_command(
     {
         vol.Required(TYPE): "zwave_js/heal_node",
-        vol.Required(ENTRY_ID): str,
-        vol.Required(NODE_ID): int,
+        vol.Required(DEVICE_ID): str,
     }
 )
 @websocket_api.async_response
 @async_handle_failed_command
-@async_get_entry
+@async_get_node
```

Comment:
Same question as above but for healing a node.

Reply:
same as above. If the first is true this is definitely true